src/App.jsx

```diff
@@ -4,10 +4,16 @@ import Tweets from './components/Tweets'
 import RightSide from './components/RightSide'
 import defaultTweets from './assets/data/tweets.js'
 import user from './assets/data/user.js'
+import { createContext } from 'react'
+
+const TweetContext = createContext()
+const ThemeContext = createContext()
 
 function App() {
+
     const [tweets, setTweets] = useState(defaultTweets)
-    const [theme, setTheme] = useState('light');
+    const [theme, setTheme] = useState( localStorage.getItem("lastTheme") ? localStorage.getItem("lastTheme") : "light");
```

Comment:
You can simplify this down to `localStorage.getItem("lastTheme") || "light"`

That's essentially "use whatever is stored in this local storage key or if it's null, use 'light'"